### ISSUE TITLE
How to solve Fiddle:DLError

### ISSUE THREAD

**Issue by zeke-taylor** [jellymann/mittsu#100]
Every time I try and run a mittsu program this error pops up:
<a href="https://ibb.co/1b1ThPK"><img src="https://i.ibb.co/KmdWRf7/Screenshot-2021-02-24-134325.png" alt="Screenshot-2021-02-24-134325" border="0"></a>
And then it immediately closes does anybody know how to fix this so that it'll work.

**Comment by jellymann** [comment]
Have you tried setting the environment variable `MITTSU_LIBGLFW_PATH` to point to the glfw3 dll file?

**Comment by danpish** [comment]
@danini-the-panini I tried it. Didn't work.
I tried setting it in advance system settings and it didn't work. I tried setting both `MITTSU_LIBGLFW_FILE`  and `MITTSU_LIBGLFW_PATH` in both cmd and advance system settings with no luck.
I also tried copying glfw3.dll to msys2 crt64 bin folder, ruby's bin folder, project root folder and also tried renaming "GLFW3.dll" to "glfw3.dll" in opengl-bindings glfw.rb file. But I think these steps were useless anyway.

**Comment by danpish** [comment]
It works! my path was messed up.
I tried installing mittsu in jruby instead. It didn't work however it gave me more useful debug info ( `Could not open library <broken-path> : The specified procedure could not be found.`). My path was messed up so I changed it and it still didn't work but at least it found the glfw3.dll.
with glfw3.dll correctly in place opening the file with normal ruby successfully opens a window. It's weird because I'm sure I got the path right the first time.

**Comment by danini-the-panini** [comment]
Yeah i don't think JRuby works properly yet. I think there's a branch where i was experimenting with Charles Nutter
